streamly.cabal

```
@@ -3,33 +3,31 @@ name:               streamly
 version:            0.7.2
 synopsis:           Data flow programming and declarative concurrency
 description:
-  Streamly is a framework for modular data flow based programming and
-  declarative concurrency.  Powerful stream fusion framework in streamly
-  allows high performance combinatorial programming even when using byte level
-  streams. Key features include:
+  <https://streamly.composewell.com Streamly> is a framework to build
+  reliable and scalable software systems from modular building blocks
+  using data flow programming (streaming) and declarative concurrency.
+  Powerful stream fusion framework in streamly allows high performance,
+  modular combinatorial programming. Key features include:
   .
-  * Performance at par with C
-  * Simple API, similar to standard Haskell lists.
-  * Declarative, auto scaled concurrency
-  * Simple, yet powerful abstractions
-      * Unifies streams, arrays and parsers
-      * Obviates the need for text and bytestring
-      * Covers the functionality of list-t and logict
-      * Covers all Data.List combinators
-      * Supports numerous time domain combinators
-      * Interworks with other streaming libraries
+  * Performance at par with C (<https://github.com/composewell/streaming-benchmarks Benchmarks>)
```

```suggestion
  * Performance on par with C (<https://github.com/composewell/streaming-benchmarks Benchmarks>)
```
Does `at` also work here?

---

`on` is correct I think.